js/dialog.js

```diff
@@ -13,9 +13,24 @@ export async function setupDetailsDialog() {
   // A11yDialog handles toggling accessibility properties when the dialog shows/ hides,
   // as well as closing on esc, clicking outside of the dialog, etc.
   DETAILS_DIALOG_A11Y = new A11yDialog(document.getElementById("card-details"));
+
+  // animation events
+  let dialogCard = DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0];
+  dialogCard.addEventListener("animationend", function() {
+    dialogCard.classList.remove("animated", "unclickable");
+  }, false);
+  dialogCard.addEventListener("mouseover", animateCard);
+  dialogCard.addEventListener("click", animateCard);
+
+  // animation reset
+  DETAILS_DIALOG_A11Y.on('hide', function (element, event) {
+    console.log("ayo?");
```

Comment:
I laughed out loud, peak tempura coding mood - delete console.log

Reply:
AAAA I thought I deleted that! Maybe I did an 'undo' and forgot to remove it. I have removed it now.